src/drawtypes/label.cpp

```diff
@@ -73,6 +82,28 @@ namespace drawtypes {
     else
       text = conf.get<string>(section, name, def);
 
```

Comment:
That's a much neater way to do it. If you don't mind, could you show how `%token:min%` fails? While testing it, I didn't seem to encounter any errors.